data/virt_autotest/setup_dns_service.sh

```diff
@@ -135,6 +182,10 @@ EOF
 dns_config_file="/etc/named.conf"
 dns_config_file_tmp="/etc/named.conf.tmp"
 dns_service_name="named"
+cat ${dns_config_file} | grep "#$0"
+if [ `echo $?` == 0 ];then
+	mv ${dns_config_file}.orig ${dns_config_file}
+fi
 cp ${dns_config_file} ${dns_config_file}.orig
```

Comment:
Would you please give some comment for why or what purpose for this dns config file recovery.